pkg/install/install.go

```diff
@@ -121,6 +121,12 @@ func RunInstall(cmd *cobra.Command, args []string) {
 		log.Printf("")
 	}
 
+	// wait for the operators to be ready before creating the system
```

Comment:
Do we have to have this part now? Usually, I break the install when waiting for the system to be Ready, but now breaking the install on wait won't create the system. 
Also, should(can) we add no-wait option support here as well?